crates/uv-resolver/src/pubgrub/dependencies.rs

```diff
@@ -185,19 +188,13 @@ fn to_pubgrub(
     urls: &Urls,
     locals: &Locals,
 ) -> Result<(PubGrubPackage, Range<Version>), ResolveError> {
-    match requirement.version_or_url.as_ref() {
-        // The requirement has no specifier (e.g., `flask`).
-        None => Ok((
-            PubGrubPackage::from_package(requirement.name.clone(), extra, urls),
-            Range::full(),
-        )),
-
-        // The requirement has a specifier (e.g., `flask>=1.0`).
-        Some(VersionOrUrl::VersionSpecifier(specifiers)) => {
+    match &requirement.source {
+        RequirementSource::Registry { version, .. } => {
+            // TODO(konsti): Index
```

Comment:
What do this mean? We need to include way more context in these comments please!